src/tuned_models.jl

```diff
@@ -379,18 +381,18 @@ function assemble_events(metamodels,
          color = :yellow)
     ch = Channel{Bool}(min(1000, length(partitions)) )
    
-    verbosity < 1 || update!(p,0)
       
     @sync begin
         # printing the progress bar
-        verbosity < 1 || @async begin
-                              while take!(ch)
-                                p.counter +=1 
-                                ProgressMeter.updateProgress!(p)
-                              end
+        verbosity < 1 || begin
+                          update!(p,0)
+                         @async while take!(ch)
+                            p.counter +=1 
+                            ProgressMeter.updateProgress!(p)
+                          end
                         end
     # One tresampling_machine per task
-    machs = [resampling_machine, [machine(Resampler(model= resampling_machine.model.model,
+    machs = [resampling_machine, [Machine{M}Resampler(model= resampling_machine.model.model,
                       resampling    = resampling_machine.model.resampling,
```

Comment:
This looks a little dangerous. The raw constructor `Machine{M}` should not be considered to have a stable API (and there is a typo here).

Reply:
just wanted to avoid the unnecessary checks that happens when the `machine` method is called since it's been called in first call to `machine` (I could revert if you don't like this?). I don't see the typo

Reply:
I've just checked the machines refactor PR and this will break the change you have here. (All data must be wrapped in a source node.) I suggest you revert to using `machine`. We can open an issue to add `checks=false` keyword constructor to the `machine` constructor, which we can use here later. 

But for now, let's revert.